Tests/scs-compatible.yaml

```diff
@@ -27,6 +27,7 @@ iaas:
         url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Standards/scs-0100-v2-flavor-naming.md
         check_tools:
           - executable: ./iaas/flavor-naming/flavor-names-openstack.py
+            args: "--accept-old-mand"
```

Comment:
We need to revert this line as we have chosen another approach: v2 compliance does mean that all mandatory v2 flavor names need to be present and we have instead chosen to expose both v1 and v2 testing results. As this is a (minor) incompatibility b/w v1 and v2 standards, better make this very visible.